tools/xtb/xtb_molecular_optimization.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="xtb_molecular_optimization" name="xtb molecular optimization" version="0.1.0+galaxy0" profile="21.05">
+       <description>Semiempirical quantum mechanical molecular optimization method</description><creator>
+        <person
+            givenName="Wudmir"
+            familyName="Rojas"
+            url="https://github.com/wverastegui"
+            identifier="0000-0001-7036-9987" />
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="6.6.1">xtb</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln '${coordinates}' 'mol.${coordinates.ext}' &&
+        xtb 'mol.${coordinates.ext}' --opt '${opt_level}' && 
+        cp 'xtbopt.${coordinates.ext}' '${output}'
+    ]]></command>
+    <inputs>
+        <param label="Atomic coordinates file" type="data" name="coordinates" format="xyz,coord" help="Initial atomic coordinates file to be optimized."/>
+        <param type="select" label="Optimization Levels" name="opt_level" help="Geometry optimization convergence levels." >
+            <option value="crude">crude</option>
+            <option value="tight">tight</option>
+            <option value="extreme">extreme</option>
+        </param>
```

Comment:
Please add the additional options.

Reply:
Added opt levels options ([53c0ffa](https://github.com/wverastegui/galaxytools/commit/53c0ffa223abe64fdf14b43245e1637c1d9ef464))